testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -330,12 +268,11 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
             .build().output
 
         then:
-        output.contains("Gradle Enterprise plugin has been disabled as it is incompatible with the isolated projects feature")
+        output.contains("Gradle Enterprise plugin $version has been disabled as it is incompatible with Isolated Projects feature. Upgrade to Gradle Enterprise plugin 3.15 or newer to restore functionality.")
```

Comment:
```suggestion
        output.contains("Gradle Enterprise plugin $version has been disabled as it is incompatible with Isolated Projects. Upgrade to Gradle Enterprise plugin 3.15 or newer to restore functionality.")
```

🤔 Not sure what we want the message to be, though for me "incompatible with Isolated Projects feature" sounds strange. I'd either use "incompatible with the Isolated Projects feature" or "incompatible with Isolated Projects". WDYT?

Reply:
I don't feel strongly, but the shorter message seems better.